### PR TITLE
Add command for automating local setup of Edly Multisite Devstack

### PR DESCRIPTION
**Pull request by mfarhan943**
Description:
To enhance the devstack process, we aim to automate the local setup of Edly Multisite Devstack.

JIRA:
[EDLY-5506](https://edlyio.atlassian.net/browse/EDLY-5506)

[EDLY-5506]: https://edlyio.atlassian.net/browse/EDLY-5506?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by taimoor-ahmed-1 — APPROVED** [review]

File: ecommerce/core/management/commands/setup_ecommerce_service.py [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+                    "OSCAR_DEFAULT_CURRENCY": "USD",
+                    "OSCAR_DEFAULT_CURRENCY_SYMBOL": "$",
+                    "CORS_ORIGIN_WHITELIST": ["https://apps.edx.devstack.lms:3030"],
+                    "OSCAR_FROM_EMAIL": "<from email address>",
+                    "SESSION_COOKIE_DOMAIN": ".edx.devstack.lms",
+                    "LANGUAGE_CODE": "en",
+                    "EDLY_WORDPRESS_URL": "http://wordpress.edx.devstack.lms/",
+                    "FRONTEND_LOGOUT_URL": "http://edx.devstack.lms:18000/logout",
+                    "PAYMENT_PROCESSOR_CONFIG": {
+                        "red": {
```
Comment (marslanabdulrauf) [inline ecommerce/core/management/commands/setup_ecommerce_service.py:85]: shouldn't it be `edly` ? or are we using something else ?

File: ecommerce/core/management/commands/setup_ecommerce_service.py [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+                "PLATFORM_NAME": "http://wordpress.edx.devstack.lms/",
+                "COLORS": {
+                    "primary": "#dd1f25",
+                    "secondary": "#dd1f25"
+                },
+                "EDLY_COPYRIGHT_TEXT": "Edly Copy Rights. All rights reserved for red site.",
+                "PANEL_NOTIFICATIONS_BASE_URL": "http://panel.edx.devstack.lms:9999/",
+                "CONTACT_MAILING_ADDRESS": "Edly 25 Mohlanwal Road, Westwood Colony Lahore, Punjab 54000",
+                "SERVICES_NOTIFICATIONS_COOKIE_EXPIRY": "900",
+                "GTM_ID": "GTM-M69F9BL",
```
Comment (marslanabdulrauf) [inline ecommerce/core/management/commands/setup_ecommerce_service.py:113]: This is not personal, right ? we can always add dummy GTM_ID I guess.

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 1 above)
Comment (mfarhan943) [reply ecommerce/core/management/commands/setup_ecommerce_service.py:85]: I have copied this from the Confluence document.

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 2 above)
Comment (mfarhan943) [reply ecommerce/core/management/commands/setup_ecommerce_service.py:113]: No, I have copied this value from the Confluence document.

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 1 above)
Comment (marslanabdulrauf) [reply ecommerce/core/management/commands/setup_ecommerce_service.py:85]: replace it with `edly`

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 2 above)
Comment (marslanabdulrauf) [reply ecommerce/core/management/commands/setup_ecommerce_service.py:113]: replace it with dummy value like `GTM-XXXXXX`

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 1 above)
Comment (mfarhan943) [reply ecommerce/core/management/commands/setup_ecommerce_service.py]: Done

File: ecommerce/core/management/commands/setup_ecommerce_service.py (hunk 2 above)
Comment (mfarhan943) [reply ecommerce/core/management/commands/setup_ecommerce_service.py]: Done

**Review by marslanabdulrauf — APPROVED** [review]

**Pull request merged**